content/en/docs/tasks/debug-application-cluster/debug-application.md

```diff
@@ -142,10 +142,10 @@ kubectl get pods --selector=name=nginx,type=frontend
 to list pods that match this selector.  Verify that the list matches the Pods that you expect to provide your Service.
 
 If the list of pods matches expectations, but your endpoints are still empty, it's possible that you don't
```

Comment:
This whole paragraph is wrong except in a relatively rare edge case.  Defining the pod `ports[]` including `containerPort` is totally optional and not defining a port will not prevent a service from forwarding to it.

The rare edge case is when an service specifies a port name (rather than number) as `targetPort`.

Reply:
I would strike this paragraph